packages/api/src/utils.ts

```diff
@@ -0,0 +1,34 @@
+import { WS } from '@deriv/shared';
+import { TSocketAcceptableProps, TSocketEndpointNames, TSocketResponseData } from '../types';
+
+export const send = async <T extends TSocketEndpointNames>(
+    name: T,
+    ...props: TSocketAcceptableProps<T, true> | TSocketAcceptableProps<T>
+): Promise<TSocketResponseData<T>> => {
+    const request = props[0];
+    if (request && 'options' in request) delete request?.options;
```

Comment:
Why do we need `options`? 😅

Reply:
@balakrishna-deriv I'm bypassing the `useQuery` options to our custom hook so devs can use them to control the behavior of the hook.

<img width="566" alt="image" src="https://user-images.githubusercontent.com/72082844/227099533-ac4825e1-7233-425d-a3e9-915215812811.png">

Reply:
Can we make it a third optional parameter in the useFetch method instead of adding it to the request?

Reply:
@balakrishna-deriv I tried but couldn't get it working, Because some endpoints don't have any props so the type of the second argument will be `[undefiled?]` which makes TS complain if we pass anything so devs can know this endpoint doesn't accept anything.

Let me spend some more time on it and see if I can find a way 🙇🏻

Reply:
Thanks @farzin-deriv. You are a genius 🧙